### PR TITLE
Moderate readme

### PR DESCRIPTION
**Pull request by tusqasi**
possibly correct some minor things

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,116 +1,124 @@
...
+
+## How To Contribute
+
+You will have to use some basic git commands for basic contributions in any
+opensource projects !
+
+First you need to clone whole repo by
+
+```bash
 git clone
```
Comment (amino19) [inline README.md:55]:
```suggestion
git clone <forked-repo-url>
```

File: README.md [hunk 2]
```diff
@@ -1,116 +1,124 @@
...
-``` 
+```
 
 Then you have to make desired changes in the repository,
-after making the changes you will have to add upstream to main repository 
+after making the changes you will have to add upstream to main repository
 
-```md
+```bash
```
Comment (amino19) [inline README.md:61]: @tusqasi : Need to remove `bash` & revert the commit here!

File: README.md [hunk 3]
```diff
@@ -1,116 +1,124 @@
...
 git push origin branch-name
 ```
 
-## How you get points 
-Submit the Merged PR link into the provided google form of open-source challenge. After evaluation of your submission, points will be awarded according to the difficulty level of the commit. Easy Level commits, Moderate Level commits and Hard Level commits will contain 1 point, 2 points and 3 points respectively.
-
-We will consider only one merged PR from a particular repository,we will not be allotting any points for other PR's in the same repository,even if it is merged.
+## How you get points
```
Comment (amino19) [inline README.md:102]: @tusqasi : Try to remove spaces below `#`, `##` & `###` :slightly_smiling_face:

**Review by amino19 — CHANGES_REQUESTED** [review]

**Comment by amino19** [comment]
@tusqasi : LGTM! :partying_face: 

@ghulamyazdani : Pl take this to your end!

**Comment by ghulamyazdani** [comment]
@tusqasi could you please make a new branch and commit to those ? this is conflicting with the recent commit!!

**Comment by ghulamyazdani** [comment]
@tusqasi could you please make a new branch and commit to those ? this is conflicting with the recent commit!!

**Comment by tusqasi** [comment]
Sure. I'll close this PR then.

**Pull request closed without merging**